content/departments/product-engineering/engineering/cloud/delivery/managed/oidc_site_admin.md

```diff
@@ -0,0 +1,103 @@
+# Purpose and Background
+
+An increasing number of customers have tight security controls and requirements for Sourcegraph Managed Instances. Sourcegraph Teammates may have the need for Site Admin access to a customer's Managed Instance. This was accomplished in the past by using a shared service user account using the `built-in` auth provider. This auth provider method lacks two-factor authentication (2FA) and has been considered unacceptable by a few customers.
```

Comment:
The framing on this paragraph feels a bit negative / scary. It could be interpreted as "our current signup method is not secure enough". @dan-mckean Do you have any recommendations here?

Reply:
I can remove this bit and just keep the latter half 🤷